runtime/src/test/kotlin/com/livefront/sealedenum/CreateSealedEnumFromEnumTests.kt

```diff
@@ -54,7 +54,6 @@ class CreateSealedEnumFromEnumTests {
                     ComparatorConfig(AlphaEnum.DELTA, AlphaEnum.GAMMA, 1),
                     ComparatorConfig(AlphaEnum.DELTA, AlphaEnum.DELTA, 0)
                 ).map { Arguments.of(it) }.stream()
-
```

Comment:
The latest version of detekt runs on `test` files be default. This was the only thing it picked up, so I don't see a reason to disable that.

Reply:
👍